docs/source/adventure/project.rst

```diff
@@ -104,6 +106,410 @@ stick with a simple text-based intro. Try to recreate the following:
+
+            @override_animation(Uncreate)
+            def _uncreate_vec_unwrite_label(self) -> AnimationGroup:
+                return AnimationGroup(
+                    Uncreate(self.vector),
+                    Unwrite(self.label),
+                    lag_ratio=0
+                )
+
+    class GeometricAddition(Scene):
+        def construct(self):
+            title = Text("Now let's take a look at it geometrically")
+            self.play(Write(title))
+            self.wait(2)
+            self.play(Unwrite(title))
+
+            plane = NumberPlane()
+
+            sum_point = (2, -1, 0)
+
+            v1 = VectorGroup(
+                ORIGIN,
+                (2, 2, 0),
+                r"\boldsymbol{\vec{v}_1}",
+                RED,
+                direction=UP,
+                plane=plane
+            )
+
+            v2 = VectorGroup(
+                ORIGIN,
+                (0, -3, 0),
+                r"\boldsymbol{\vec{v}_2}",
+                YELLOW,
+                direction=LEFT,
+                plane=plane
+            )
+
+            v1moved = VectorGroup(
+                (0, -3, 0),
+                sum_point,
+                r"\boldsymbol{\vec{v}_1}",
+                v1.vector.get_color(),
+                plane=plane
+            )
+
+            v2moved = VectorGroup(
+                (2, 2, 0),
+                sum_point,
+                r"\boldsymbol{\vec{v}_2}",
+                v2.vector.get_color(),
+                plane=plane
+            )
+
+            sum_vec = VectorGroup(
+                ORIGIN,
+                sum_point,
+                r"\boldsymbol{\vec{v}_1}+\boldsymbol{\vec{v}_2}",
+                ORANGE,
+                direction=DOWN,
+                plane=plane
+            )
+
+            self.play(Create(plane), Create(v1))
+            self.wait(0.5)
+            self.play(Create(v2))
+            self.wait()
+
+            # animate movement of vectors
+            self.play(
+                Succession(
+                    ReplacementTransform(v1.copy(), v1moved),
+                    ReplacementTransform(v2.copy(), v2moved)
+                )
+            )
+            self.wait()
+            # draw sum vector
+            self.play(Create(sum_vec))
+            self.wait()
+            self.play(*[
+                Uncreate(x)
+                for x in (
+                    plane,
+                    v1,
+                    v2,
+                    v1moved,
+                    v2moved,
+                    sum_vec
+                )
+            ])
+
+Hints
+-----
+
+Use :class:`.NumberPlane` to define the cartesian plane.
+
+Use :class:`.Arrow` for the vectors.:
+
+To make sure the label of the vector and the vector shift together, you can define a custom :class:`.VGroup` subclass.
+Take a look at the decorator :meth:`override_animation` to override the :class:`.Create` and :class:`.Uncreate` animations, it will come in handy when animating the subclass.
```

Comment:
```suggestion
Take a look at the decorator :func:`.override_animation` to override the :class:`.Create` and :class:`.Uncreate` animations, it will come in handy when animating the subclass.
```
I believe override_animation is a function so this might help the hyperlink.